Modules/_json.c

```diff
@@ -637,30 +640,27 @@ scanstring_unicode(PyObject *pystr, Py_ssize_t end, int strict, Py_ssize_t *next
     return NULL;
 }
 
-PyDoc_STRVAR(pydoc_scanstring,
-    "scanstring(string, end, strict=True) -> (string, end)\n"
-    "\n"
-    "Scan the string s for a JSON string. End is the index of the\n"
-    "character in s after the quote that started the JSON string.\n"
-    "Unescapes all valid JSON string escape sequences and raises ValueError\n"
-    "on attempt to decode an invalid string. If strict is False then literal\n"
-    "control characters are allowed in the string.\n"
```

Comment:
Wait.

This paragraph of docstring is entirely lost.  I think you shouldn't alter docstrings, except for case where you need to split out "summary line", per PEP 257.  Everything else should go to a separate pr.